### PR TITLE
[TH2-2653]. Fix messages list scrolls to attached message after next/prev chunk is loaded

### PR DESCRIPTION
**Pull request by rzvdv**
…prev chunk loaded

### CODE REVIEW

File: src/components/message/message-card-list/MessagesVirtualizedList.tsx [hunk 1]
```diff
@@ -75,7 +68,10 @@ const MessagesVirtualizedList = (props: Props) => {
 	React.useEffect(() => {
 		if (scrolledIndex !== null) {
 			raf(() => {
-				virtuoso.current?.scrollToIndex({ index: scrolledIndex.valueOf(), align: 'center' });
+				virtuoso.current?.scrollToIndex({
+					index: scrolledIndex,
+					behavior: 'smooth',
```
Comment (rzvdv) [inline src/components/message/message-card-list/MessagesVirtualizedList.tsx:73]: When i tested it worked not stable, i don't think we need that

File: src/components/message/message-card-list/MessagesVirtualizedList.tsx [hunk 2]
```diff
@@ -30,14 +30,7 @@ interface Props {
...
 	itemRenderer: (index: number, message: EventMessage) => React.ReactElement;
-	/*
-		Number objects is used here because in some cases (eg one message / action was 
-		selected several times by different entities)
-		We can't understand that we need to scroll to the selected entity again when
-		we are comparing primitive numbers.
-		Objects and reference comparison is the only way to handle numbers changing in this case.
-	*/
-	scrolledIndex: Number | null;
+	scrolledIndex: number | null;
```
Comment (rzvdv) [inline src/components/message/message-card-list/MessagesVirtualizedList.tsx:33]: It should be a Number object. There are some edge cases when previous scrolledIndex equals to new scrolledIndex, so if it was a number primivite type we couldn't detect it

File: src/stores/messages/MessagesDataProviderStore.ts [hunk 3]
```diff
@@ -137,6 +137,12 @@ export default class MessagesDataProviderStore {
 				}
 			}
 
+			const { selectedMessageId, scrollToMessage } = this.messagesStore;
+			this.searchChannelNext.onStop = () =>
```
Comment (rzvdv) [inline src/stores/messages/MessagesDataProviderStore.ts]: this code should be deleted

Comment (VitalyMashanov) [reply src/stores/messages/MessagesDataProviderStore.ts]: when you're picking some event, message list is not scrolling to attached message without this part

File: src/components/message/message-card-list/MessagesVirtualizedList.tsx (hunk 2 above)
Comment (VitalyMashanov) [reply src/components/message/message-card-list/MessagesVirtualizedList.tsx:33]: when you choose another event with the same attached message
messages list is reloading
is there any other edge cases?

File: src/stores/messages/MessagesDataProviderStore.ts (hunk 3 above)
Comment (VitalyMashanov) [reply src/stores/messages/MessagesDataProviderStore.ts]: for example

Comment (rzvdv) [reply src/stores/messages/MessagesDataProviderStore.ts]: then we should develop new appropriate solution to scroll to selected message

**Comment by rzvdv** [comment]
fixed and merged in #275

**Comment by rzvdv** [comment]
fixed and merged in #275

**Pull request closed without merging**